### ISSUE TITLE
How does the logit_scale vary while training , i noitced that in my case it starts from the 14.28(1/0.07) and then just goes down and towards the end of the training it reaches 1

### ISSUE THREAD

**Issue by Akshay1-6180** [mlfoundations/open_clip#815]
<img width="443" alt="Screenshot 2024-02-13 at 1 31 12 PM" src="https://github.com/mlfoundations/open_clip/assets/65217827/bb7eeaa9-7f46-49e5-b684-688ee7daa023">
<img width="439" alt="Screenshot 2024-02-13 at 1 31 25 PM" src="https://github.com/mlfoundations/open_clip/assets/65217827/f20600ff-b00a-4acd-86d1-53457546a1d9">

I am running clip on my own dataset and noticed this where the logit_scale converges to 1.
Is this a good behavior to expect , i noticed that the loss becomes constant during this time.
I know that a higher logit_scale amplifies differences between the logits, making the softmax output distribution sharper and thus making the model more confident in its most likely predictions , does the model lowering the logits mean the model is becoming less confident or the model is getting confused 
reducing the learning rate resolves this issue but it will start converting towards a value lower than 14 (mostly btw 6-8).Not sure what conclusion i can make from this.
I use adamw optimizer with a VIT B vision model , Bert text encoder and weight decay is 0.1 , eps is 1e-8,betas=[0.9,0.999]

**Comment by rom1504** [comment]
https://wandb.ai/rom1504/open-clip/reports/xlm-roberta-base-B-32--VmlldzoyOTQ5OTE2 find here one fairly normal clip run

You should see logit scale going to 1, loss decreasing, lr decreasing and accuracy increasing, all fairly in sync.

**Comment by Akshay1-6180** [comment]
Thanks @rom1504 Thanks for the logs , isnt the logit_scale going towards 100 as the loss decreases in this case , not to 1

**Comment by rwightman** [comment]
yes, should end up around 100 by end of training. There is usually something wrong with the training setup, most likely data or optimization hparams if that doesn't happen.
